tests/core/pyspec/eth2spec/test/phase0/genesis/test_initialization.py

```diff
@@ -216,3 +220,36 @@ def test_initialize_beacon_state_random_valid_genesis(spec):
     assert spec.is_valid_genesis_state(state)
 
     yield 'state', state
+
+
+@with_phases([PHASE0])
+@spec_test
+@single_phase
+@with_presets([MINIMAL], reason="too slow")
+def test_genesis_block_creation(spec):
```

Comment:
ditto.

This test case should be moved to `tests/core/pyspec/eth2spec/test/phase0/unittests/test_genesis.py` since it doesn't create test vectors.